janusgraph-hadoop-parent/janusgraph-hadoop-core/src/test/resources/org/janusgraph/hadoop/formats/graphson/incremental-naive-cerberus-load.groovy

```diff
@@ -12,20 +12,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-def JanusGraphVertex getOrCreateVertex(faunusVertex, graph, context, log) {
-    String uniqueKey = "name";
-    Object uniqueValue = faunusVertex.value(uniqueKey);
-    Vertex janusgraphVertex;
+JanusGraphVertex getOrCreateVertex(faunusVertex, graph, context, log) {
+    String uniqueKey = "name"
+    Object uniqueValue = faunusVertex.value(uniqueKey)
+    Vertex janusgraphVertex
     if (null == uniqueValue)
-      throw new RuntimeException("The provided Faunus vertex does not have a property for the unique key: " + faunusVertex);
-  
-    Iterator<Vertex> itty = graph.query().has(uniqueKey, uniqueValue).vertices().iterator();
+      throw new RuntimeException("The provided Faunus vertex does not have a property for the unique key: " + faunusVertex)
+
+    Iterator<Vertex> itty = graph.query().has(uniqueKey, uniqueValue).vertices().iterator()
     if (itty.hasNext()) {
-      janusgraphVertex = itty.next();
-      if (itty.hasNext())
-        log.info("The unique key is not unique as more than one vertex with the value {}", uniqueValue);
+      janusgraphVertex = itty.next()
+        if (itty.hasNext())
```

Comment:
(nitpick) This indent looks wrong to me. Should the `if` have the same indentation as the line above?

Reply:
Hm. Looks like it should have the same indentation level. I don't know why Intelij IDEA made this. I will fix it

Reply:
Fixed here: https://github.com/JanusGraph/janusgraph/compare/d8f481001970915363ff38a52517ee3ae963d0d1..073c7512102cdd5de723c0740406ec705b9fae6a

Reply:
Fixed other similar problems here: https://github.com/JanusGraph/janusgraph/compare/073c7512102cdd5de723c0740406ec705b9fae6a..f69337316186edf248f64b792dc5258bc9fbd8f5